lib/transactions.js

```diff
@@ -269,36 +230,51 @@ TxController.prototype.list = function(req, res) {
         return common.handleErrors(err, res);
       }
 
-      var blockInfo = self.node.services.bitcoind.getBlockIndex(block.hash);
-      var txs = block.transactions;
-      var totalTxs = txs.length;
+      self.node.services.bitcoind.getBlockHeader(block.hash, function(err, blockInfo) {
```

Comment:
Handled in: https://github.com/bitpay/insight-api/pull/467/commits/b446f6733c3f71ea4a983d10e5f4ce4f15f97c68